src/include/wt_internal.h

```diff
@@ -137,6 +137,8 @@ struct __wt_ckpt_block_mods;
 typedef struct __wt_ckpt_block_mods WT_CKPT_BLOCK_MODS;
 struct __wt_ckpt_connection;
 typedef struct __wt_ckpt_connection WT_CKPT_CONNECTION;
+struct __wt_ckpt_handle;
+typedef struct __wt_ckpt_handle WT_CKPT_HANDLE;
```

Comment:
Should I make this plural?

```suggestion
struct __wt_ckpt_handles;
typedef struct __wt_ckpt_handles WT_CKPT_HANDLES;
```